custom/icds/translations/integrations/client.py

```diff
@@ -37,9 +37,13 @@ def get_resource_slugs(self, version):
         """
         :return: list of resource slugs corresponding to version
         """
-        return [r['name']
-                for r in self.list_resources().json()
-                if r['name'].endswith("v%s" % version)]
+        all_resources = self.list_resources().json()
+        if version:
+            return [r['slug']
+                    for r in self.list_resources().json()
+                    if r['slug'].endswith("v%s" % version)]
+        else:
+            return [r['slug'] for r in all_resources]
```

Comment:
Changed to using slug and not name because that's what it should be. Till now name and slug was the same so this has not caused any issue but its still a possibility for them to differ. So updated to use slug instead.